README.md

```diff
@@ -18,6 +18,34 @@ That's why the goal of this project is to make it easy to cleanly
 install Sharness support, so that Sharness files are separated from
 other files and can easily be installed and updated.
 
+## Usage
+
+Clone this repo to a temporary directory and run `sharnessify.sh` with the path
+to your project:
+
+```sh
+$ cd /tmp
+
+$ git clone git://github.com/chriscool/sharnessify
+
+$ cd sharnessify
+
+$ sh sharnessify.sh ~/dev/my-project
+
+$ cd ~/dev/my-project
```

Comment:
But maybe your explanations are simpler for someone who don't know git well.
So I am ok to merge them as is.